clusters/app.ci/prow/03_deployment/pj-rehearse-plugin.yaml

```diff
@@ -65,6 +65,7 @@ items:
           - --github-endpoint=https://api.github.com
           - --github-token-path=/etc/github/oauth
           - --config-path=/etc/config/config.yaml
+          - --gsc-bucket=test-platform-results
```

Comment:
There's also `gcs-browser-prefix` .

Reply:
I think this one will be covered by the alias right?